compiler/integration-tests/test/node/smart_contract_verifier.test.ts

```diff
@@ -46,7 +46,7 @@ test_cases.forEach((testInfo) => {
     const inputs = toml.parse(prover_toml);
     const { witness } = await program.execute(inputs);
 
-    const backend = new UltraPlonkBackend(noir_program.bytecode, {}, { recursive: false });
+    const backend = new UltraHonkBackend(noir_program.bytecode, {}, { recursive: false });
     const proofData = await backend.generateProof(witness);
```

Comment:
You might need to do `proofData.proof.slice(4)` before passing to the contract

See [11829](https://github.com/AztecProtocol/aztec-packages/issues/11829) - hopefully can be fixed before next release

Reply:
Thanks, I've done this and am now getting a sumcheck failure. Not sure if it's related to the issues being reported in slack.

Reply:
Ah you need to add `{ keccak: true }` as well

Reply:
11829 seems to be an easy fix. I will do a PR